README.md

```diff
@@ -29,7 +29,6 @@ You can find [the documentation here](https://imitation.readthedocs.io/en/latest
 - Python 3.8+
 - (Optional) OpenGL (to render Gym environments)
 - (Optional) FFmpeg (to encode videos of renders)
-- (Optional) MuJoCo (follow instructions to install [mujoco_py v1.5 here](https://github.com/openai/mujoco-py/tree/498b451a03fb61e5bdfcb6956d8d7c881b1098b5#install-mujoco))
```

Comment:
Just to doublecheck, this is unnecessary since this is now entirely handled by this line in setup.py, right?
https://github.com/HumanCompatibleAI/imitation/blob/de589d46ee81e1c83635d3b4865884abf8960f44/setup.py#L223C1-L223C1

Reply:
Yes!